src/vcSettings.h

```diff
@@ -59,6 +59,21 @@ enum
   vcMetadataMaxLength = 256,
 };
 
+enum vcMapTileServerFlag
+{
+	vcMTSF_Unknown,
+	vcMTSF_Slippy,
+	vcMTSF_Google,
+
+	vcMTSF_Count
+};
+
+const char g_vcMapTileServerName[vcMTSF_Count][64] = {
+	{ "unknown" },
+	{ "slippy" },
+	{ "google" }
+};
```

Comment:
This is a weird format that is inconsistent with the other strings. We generally use:

```
const char *g_vcMapTileServerName[] = { "unknown", "slippy", "google" };
UDCOMPILEASSERT(udLengthOf(g_vcMapTileServerName) == vcMTSF_Count, "Update array");
```

(sometimes to split individual values on each line for larger enums)